src/LiveComponent/composer.json

```diff
@@ -29,7 +29,9 @@
         "php": ">=8.0",
         "symfony/property-access": "^5.4|^6.0",
         "symfony/serializer": "^5.4|^6.0",
-        "symfony/ux-twig-component": "^2.5"
+        "symfony/twig-bundle": "^5.4|^6.0",
```

Comment:
I think `TwigBundle` is not required here (nothing imported from `TwigBundle` in `src` only from `twig`).

Reply:
Like the component it requires the specific service `twig` and so has a requirement to twig bundle itself.

Reply:
Ahh do you think about this one? https://github.com/symfony/ux/blob/4bd27efca1851bdaf1574c922dd8fa437229aa36/src/LiveComponent/src/DependencyInjection/LiveComponentExtension.php#L131

If, we add the bundle this can be removed https://github.com/symfony/ux/blob/4bd27efca1851bdaf1574c922dd8fa437229aa36/src/LiveComponent/src/DependencyInjection/LiveComponentExtension.php#L51-L55 

And there are other missing dependencies too, like dependency-injection, router, http-kernel, ... don't we should add those too?

Reply:
Yes, that is what currently failing the installation inside a empty skeleton. I personally would if they are required, but not sure, wait what @weaverryan think about that.